geomesa-spark/geomesa-spark-sql/src/test/scala/org/locationtech/geomesa/spark/SparkSQLGeometryProcessingFunctionsTest.scala

```diff
@@ -43,31 +43,44 @@ class SparkSQLGeometryProcessingFunctionsTest extends Specification with LazyLog
 
 
     "st_antimeridianSafeGeom" >> {
-      val geom = "st_geomFromWKT('POLYGON((-190 50, -190 60, -170 60, -170 50, -190 50))')";
-      val decomposed = sc.sql(s"select st_antimeridianSafeGeom($geom)").collect.head.get(0).asInstanceOf[Geometry]
-      decomposed mustEqual WKTUtils.read(
-        "MULTIPOLYGON (((-180 50, -180 60, -170 60, -170 50, -180 50)), ((180 60, 180 50, 170 50, 170 60, 180 60)))")
+      "should handle nulls" >> {
+        sc.sql("select st_antimeridianSafeGeom(null)").collect.head(0) must beNull
+      }
+
+      "should split a geom that spans the antimeridian" >> {
```

Comment:
Should we verify that the st_antimeridianSafeGeom code is a no-op away from the anti-meridian?  (Separate test?)